proto/eth/v2/custom.go

```diff
@@ -0,0 +1,51 @@
+package eth
+
+import (
+	"bytes"
+	"math/bits"
+)
+
+const (
+	NextSyncCommitteeIndex = uint64(55)
+	FinalizedRootIndex     = uint64(105)
```

Comment:
do these consts have to be exported?

Reply:
They are used by the light client store later in the series.